### PR TITLE
Proxy server compatibility changes

### PR DESCRIPTION
**Pull request by Atr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js --mode development --watch",
```
Comment (ivan-cordoba) [inline package.json]: I would recommend changing this to remove the watch and dev or creating two different commands.One for dev which can include the --watch and one for standard build which does not include --watch

File: package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
...
     "start": "nodemon server/server.js",
```
Comment (ivan-cordoba) [inline package.json]: Similar to 'build' I would reccommend having a build-dev which uses nodemon and a build which uses node

File: package.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
   "scripts": {
```
Comment (ivan-cordoba) [inline package.json:10]: you can add a 'postinstall' script that will run immediately after npm install finishes. You can add anything that must run to here, for example I added my webpack build to the 'postinstall' script

**Review by ivan-cordoba — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Atr) [reply package.json:10]: Good to know, thanks!

File: package.json (hunk 1 above)
Comment (Atr) [reply package.json]: Great idea.

File: package.json (hunk 2 above)
Comment (Atr) [reply package.json]: Gotcha!  Thanks Ivan : )

**Review by ivan-cordoba — APPROVED** [review]

**Pull request merged**